public/index.html

```diff
@@ -1,8 +1,10 @@
 <!DOCTYPE html>
 <head>
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
```

Comment:
I don't think you need these scripts since webpack is going to transpile your .jsx files to your public/index.html